openacademy/models/openacademy_session.py

```diff
@@ -1,13 +1,80 @@
-from odoo import models
-from odoo import fields
+from odoo import models, fields, api, exceptions
+from datetime import timedelta
+import time
 
 
 class Session(models.Model):
     _name = 'openacademy.session'
+    _description = 'Model to manage and schedule course sessions'
 
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=fields.Date.today)
+    datetime_test = fields.Datetime(default=fields.Datetime.now)
     duration = fields.Float(digits=(6, 2), help="Duration in days")
     seats = fields.Integer(string="Number of seats")
+    instructor_id = fields.Many2one('res.partner',
+            domain=['|', ('instructor', '=', True),
+            ('category_id.name', 'ilike', 'Teacher')])
+    course_id = fields.Many2one('openacademy.course',
```

Comment:
This change https://github.com/jjbenavaz/openacademy-repo/pull/2#discussion_r883792920 is missing here.